coalib/parsing/FilterHelper.py

```diff
@@ -49,6 +55,37 @@ def apply_filter(filter_name, filter_args, all_bears=None):
     return local_bears, global_bears
 
 
+def apply_section_filter(filter_name, filter_args, all_sections):
+    """
+    Returns sections after filtering based on ``filter_args``. It returns
+    all sections if nothing is present in ``filter_args``.
+
+    :param filter_name:
+        Name of the filter.
+    :param filter_args:
+        Arguments of the filter to be passed in.
+        For example:
+        ``['tags', ('java',)]``
```

Comment:
please use exampes from your cEP, and your own real needs.

this looks like language , not a tag which should be an arbitrary grouping which couldnt be achieved via any other means.

Reply:
I used them as examples, but I will update it.